elixir/apps/domain/test/support/fixtures/config.ex

```diff
@@ -4,7 +4,7 @@ defmodule Domain.Fixtures.Config do
 
   def configuration_attrs(attrs \\ %{}) do
     Enum.into(attrs, %{
-      clients_upstream_dns: [%{address: "1.1.1.1"}]
+      clients_upstream_dns: [%{type: "ip", address: "1.1.1.1"}]
```

Comment:
Would it make sense to add a few more valid examples here to increase coverage in our integration tests?

Reply:
Possibly.  Do you have an example of what extended coverage you had in mind?  Were you thinking along the lines of having some IPs with ports appended and some that were only IPs (like what's there now)?

Reply:
Yeah maybe a couple more, like

```
2606:4700:4700::1111
8.8.8.8:853
```

Reply:
cc @conectado

Reply:
Ah, ok, yeah I don't think that should be a problem

Reply:
Updated the fixture to include the examples you gave.

Reply:
Nice, sgtm